### PR TITLE
added stuypulse calendar link, position needs to be tweaked a little, wi...

### PR DESCRIPTION
**Pull request by dknopf**
...ll do later

### CODE REVIEW

**Comment by omgimanerd** [comment]
ugly af

File: about.html [hunk 1]
```diff
@@ -130,5 +127,74 @@
...
+        <tr>
+          <td>
+            <img src='data/team/alvin_lin.jpg'>
+          </td>
+          <td>
+            Alvin Lin
+            <br>Vice President - Junior
+            <br>Head of Software Engineering
+            <br>I joined robotics because of an interest in computer programming as a career and
+I wanted to challenge myself with the difficulties that robotics would present.
```
Comment (omgimanerd) [inline about.html]: Ideally this should all be aligned with <br>

File: about.html [hunk 2]
```diff
@@ -130,5 +127,74 @@
...
+          <td>
+            <b>Daniel Knopf(Freshman)</b> - I am currently a freshman at Stuyvesant High School, and this is my first year on the Stuy robotics team, but ,my third year doing FTC. My job is website development, including this page itself.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <img src='data/team/James_yang.jpg' width="100" height="100px">
+          </td>
+          <td>
+            <b>James yang(Sophmore)</b> -Although this is my first year on the team, robotics has been a fun journey
```
Comment (omgimanerd) [inline about.html]: Sophomore?

**Comment by omgimanerd** [comment]
Crop all the images to just the shoulders and head, rotate them correctly.

File: about.html (hunk 2 above)
Comment (dknopf) [inline about.html]: What?

Comment (omgimanerd) [inline about.html]: James Yang (Sophomore)

File: about.html [hunk 3]
```diff
@@ -113,13 +110,13 @@
       <div id='footer'>
         Copyright &copy; 2014 Stuy Fission 310. All Rights Reserved.
         <div id='footer-media'>
-          <a href='http://github.com/stuyfission' />
+          <a href='http://github.com/stuyfission' >
```
Comment (omgimanerd) [inline about.html]: <a href='http://github.com/stuyfission'>

File: about.html [hunk 4]
```diff
@@ -130,5 +127,181 @@
...
+            <b>Daniel Knopf (Freshman)</b> - I am currently a freshman at Stuyvesant High School, and this is my first year on the Stuy robotics team, but ,my third year doing FTC. My job is website development, including this page itself.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <img src='data/team/James_yang_head.jpg' width="250px" height="200px">
+          </td>
+          <td>
+            <b>James yang (Sophmore)</b> - Although this is my first year on the team, robotics has been a fun journey
+in the few months since I joined. It really changed how I look on technology and teamwork as a whole.
```
Comment (omgimanerd) [inline about.html]: align all bios

**Comment by omgimanerd** [comment]
Delete all the uncropped photos, they should not have been committed. hyphenate the file name
for example
alvin_lin.jpg
kevin_yan.jpg

File: about.html [hunk 5]
```diff
@@ -130,5 +127,196 @@
...
+              <td>
+                <b>Kevin Yan
+                  <br>Head of Marketing - Junior</b>
+                <br>I joined Stuy Fission 310 as a                      marketer because of my passion for                    robotics and marketing. I was first                    introduced to robotics at a summer                    program hosted by Johns Hopkins                        Center for Talented Youth. Soon after                  that experience, my parents bought me                  a LEGO Mindstorms NXT kit. In 7th                      grade, I joined Mark Twain                            Intermediate School’s robotics team                    along with Alvin. There, I learned a                  great deal about the teamwork                          required to hold a team together. I                    also took an interest in marketing as                  well during my middle school years.                    starting with just $5, I bought and                    sold things and earned over $5000                      over these past five years. These                      experiences guided me to becoming the                  head of marketing Stuy Fission 310.
```
Comment (omgimanerd) [inline about.html]: bad

File: news.html [hunk 6]
```diff
@@ -56,6 +56,9 @@
           News
         </h1>
         <hr />
+        <a id='news-stuypulse-calendar' href='http://stuypulse.com/calendar/'>
```
Comment (omgimanerd) [inline news.html]: hyper link on h1 level

**Comment by omgimanerd** [comment]
All photos still misnamed.

**Comment by omgimanerd** [comment]
All photos still misnamed.

**Comment by omgimanerd** [comment]
All photos still misnamed.

**Comment by omgimanerd** [comment]
All photos still misnamed.

**Comment by omgimanerd** [comment]
All photos still misnamed.

**Comment by omgimanerd** [comment]
All photos still misnamed.

File: about.html [hunk 7]
```diff
@@ -130,5 +127,238 @@
...
+        <div id="about-content-divisions-engineers-popup">
+          <table>
+            <tr>
+              <td>
+                <img src='data/team/james_chin.jpg' width='250px' height="325px">
+              </td>
+              <td>
+                James Chin
+                <br>President - Junior
+                <br>Head of Mechanical & Electrical Engineering
```
Comment (omgimanerd) [inline about.html]: Use the escape sequence for &
&amp;
The ampersand is a reserved character in html.

Comment (dknopf) [inline about.html]: what is the escape sequence

**Comment by omgimanerd** [comment]
You'll have to rename the images themselves as well./

**Pull request merged**